components/azure.key.manager/src/main/java/org/wso2/azure/client/AzureADClient.java

```diff
@@ -260,9 +261,18 @@ public KeyManagerConfiguration getKeyManagerConfiguration() throws APIManagement
     @Override
     public OAuthApplicationInfo mapOAuthApplication(OAuthAppRequest oAuthAppRequest) throws APIManagementException {
         String consumerKey = oAuthAppRequest.getOAuthApplicationInfo().getClientId();
-
+        OAuthApplicationInfo clientInfo = null;
         if (StringUtils.isNotBlank(consumerKey)) {
-            OAuthApplicationInfo clientInfo = retrieveApplication(consumerKey);
+            try {
+                ClientInformation clientInformation = appClient.getApplicationByAppId(consumerKey);
+                if ( clientInformation != null ) {
```

Comment:
Can you format this code section please?

Reply:
done via https://github.com/wso2-extensions/apim-km-azure-ad/pull/8/commits/2ccef20a76c3a59530c787eb9c43dcd5b5d2e835